src/azure-cli-core/azure/cli/core/auth/msal_credentials.py

```diff
@@ -20,6 +20,7 @@
 from knack.util import CLIError
 from msal import PublicClientApplication, ConfidentialClientApplication
 
+from .constants import AZURE_CLI_CLIENT_ID
```

Comment:
As `identity.py` imports from `msal_credentials.py`, `AZURE_CLI_CLIENT_ID` is moved from `identity.py` to `constants.py` to avoid circular import (`msal_credentials.py` imports from `identity.py`).

Although circular imports can also be avoided by importing modules within a function or method, this is not as good as the current solution.